src/screens/Customers/CustomerProfile.tsx

```diff
@@ -126,35 +157,291 @@ function CustomerProfile(props: CustomerProps) {
+            <div style={{ float: 'right' }} className={classes.meterInfoContainer}>
+              <div style={{ marginLeft: '4px' }} className={classes.meterInfo}>
+                <Typography variant="body1">{meterInfo[3].label}</Typography>
+                <Typography variant="h3">
+                  {meterInfo[3].number} {meterInfo[3].unit}
+                </Typography>
+              </div>
+            </div>
+            { /* Bottom Left */ }
+            <div style={{ float: 'right' }} className={classes.meterInfoContainer}>
+              <div style={{ marginRight: '4px' }} className={classes.meterInfo}>
+                <Typography variant="body1">{meterInfo[1].label}</Typography>
+                <Typography variant="h3">
+                  {meterInfo[1].number} {meterInfo[1].unit}
+                </Typography>
+              </div>
+            </div>
+          </div>
+        </div>
+      );
+    } else if (customer.meterType == "Smart Meter") {
+      return (
+        <div>
+          <OutlinedCardList info={readingInfo} primary={false} grayed={true} />
+          <div className={classes.section}>
+            { /* Top Left */ }
+            <div style={{ float: 'left' }} className={classes.meterInfoContainer}>
+              <div style={{ marginRight: '4px' }} className={classes.meterInfo}>
+                <Typography variant="body1">{meterInfo[0].label}</Typography>
+                <Typography variant="h3">
+                  {meterInfo[0].number} {meterInfo[0].unit}
+                </Typography>
+              </div>
+            </div>
+            { /* Top Right */ }
+            <div style={{ float: 'left' }} className={classes.meterInfoContainer}>
+              <div style={{ marginLeft: '4px' }} className={classes.meterInfo}>
+                <Typography variant="body1">{meterInfo[2].label}</Typography>
+                <Typography variant="h3">
+                  {meterInfo[2].number} {meterInfo[2].unit}
+                </Typography>
+              </div>
+            </div>
+            { /* Bottom Right */ }
+            <div style={{ float: 'right' }} className={classes.meterInfoContainer}>
+              <div style={{ marginLeft: '4px' }} className={classes.meterInfo}>
+                <Typography variant="body1">{meterInfo[3].label}</Typography>
+                <Typography variant="h3">
+                  {meterInfo[3].number} {meterInfo[3].unit}
+                </Typography>
+              </div>
+            </div>
+            { /* Bottom Left */ }
+            <div style={{ float: 'right' }} className={classes.meterInfoContainer}>
+              <div style={{ marginRight: '4px' }} className={classes.meterInfo}>
+                <Typography variant="body1">{meterInfo[1].label}</Typography>
+                <Typography variant="h3">
+                  {meterInfo[1].number} {meterInfo[1].unit}
+                </Typography>
+              </div>
+            </div>
+          </div>
```

Comment:
Can each of these be made into their own component? (Maybe each component could be one of the `MeterInfoContainer`?). Looks like it's all the same besides what's getting passed into it. This'll help make the code more refactorable when we / the NPO make changes to it

Reply:
If you do choose to create a component for this, you can also use [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to display each JSX element more cleanly! [React's documentation](https://reactjs.org/docs/lists-and-keys.html) has some great examples of this. 

For your case, something like 
```
const listItems = myList.map((position) =>
    // Correct! Key should be specified inside the array.
    <MyCustomTypographyElement key={position.toString()} value={position} />
  );
```
might be useful!

Reply:
I also noticed that you're hard-coding margin-lefts and margin-rights. This approach is fine for now, but there's also a way to make a 2 by 2 grid without hard-coding! This [StackOverflow](https://stackoverflow.com/questions/24197007/how-can-i-show-three-columns-per-row) post shows how to format six plain old divs into a grid with two rows and three columns.

Reply:
Would it be ok if I made a MeterInfo component to contain all 4 conditions? I think there's a lot of subtle style changes between the meter types (ex. grayed text, links, icons) that'd be hard to generalize.

Reply:
Actually wait lemme try this first